src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,27 @@
+package codeu.chat.common;
+
+import java.io.IOException;
+
+import codeu.chat.util.Uuid;
+
+public final class ServerInfo {
+  private final static String SERVER_VERSION = "1.0.0";
+
+  public final Uuid version;
+  
+  public ServerInfo() {
+    Uuid version = null;
+    
+    try {
+      version = Uuid.parse(SERVER_VERSION);
+    } catch (IOException e) {
+      System.out.println("Could not parse version");
```

Comment:
Either throw an unchecked exception or log the error with `codeu.chat.util.Logger`.

I would throw an unchecked exception (such as an `IllegalArgumentException`) because we can only get here through programmer error, and there is no graceful way to recover. If we don't do this, then we'll probably encounter a `NullPointerException` somewhere later, which is less helpful.